projects/aca-content/src/lib/components/sidenav/components/expand-menu.component.ts

```diff
@@ -55,7 +55,7 @@ export class ExpandMenuComponent implements OnInit {
   item: NavBarLinkRef;
 
   @Output()
-  actionClicked = new EventEmitter<void>();
+  actionClicked = new EventEmitter<NavBarLinkRef>();
```

Comment:
Is that anywhere used? I think actually it's unused output as none parent uses it

Reply:
Good point, removed